.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -59,7 +58,9 @@ body:
     id: node-version
     attributes:
       label: Node.js version
-      description: Which version of Node.js are you using? Run `node --version` in your project directory and paste the output.
+      description: |
+        Which version of Node.js are you using? Run `node --version` in your project directory and paste the output.
+        If your issue is directly related to TypeScript, please include the version of TypeScript you are using as well. (`npm list typescript`)
```

```suggestion
        If you are using TypeScript include its version as well: `npm list typescript`
```
or similar, the proposed version is unnecessarily wordy

---

doesn't hurt to provide it and people are often not even sure what the issue is related to. the longer text gets the less likely people are to actually read the instruction and even consider it

---

Alright then how does this sound
```suggestion
        If you are using TypeScript, please include its version as well. (`npm list typescript`)
```
(Or maybe with npm ls, why don’t we suggest that one instead)

---

> (Or maybe with npm ls, why don’t we suggest that one instead)

the general paradigm of verbosity, knowing what things do is generally preferable, hence why we generally use `--version` and `install`, rather than the short flags.

parenthesis after the sentence generally refer to a reference so

```suggestion
        If you are using TypeScript, please include its version (`npm list typescript`) as well. 
```
or even
```suggestion
        If you are using TypeScript, please include its version (`npm list typescript`). 
```
dropping the "as well"

---

went with the first one, thanks